src/go.mod

```diff
@@ -24,6 +24,7 @@ require (
 	github.com/tidwall/pretty v1.2.1
 	github.com/tomlazar/table v0.1.2
 	github.com/vbauerster/mpb/v8 v8.1.4
+	github.com/zeebo/assert v1.1.0
```

Comment:
I believe this can also be removed since the assert package was added by mistake